.github/workflows/update-action-tag.yml

```diff
@@ -0,0 +1,46 @@
+name: Update Tag
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  update-tag:
+    name: Update Tag
+    runs-on: ubuntu-latest
+    environment: release
+
+    steps:
+      - uses: actions/create-github-app-token@v1
```

Comment:
Doesn't this action necessitate the `id-token: write` permission?

Reply:
Fixed!